recipes/python-awips/meta.yaml

```diff
@@ -0,0 +1,152 @@
+{% set name = "python-awips" %}
+{% set version = "18.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  #sha256: 61b32abccfb283fbb96f109b6d4de6b465687f9f40f3d727a113647f4a6ef760
+  url: https://github.com/Unidata/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ae331fe552a9e8e2179adbbdb98aea7f73a36fe729d69f67a077129f2571620f
+
+build:
```

Comment:
Can this package be `noarch: python`?

https://conda-forge.org/docs/meta.html#building-noarch-packages